uktrade/data-hub-frontend
# Reactify edit OMIS quote information form

## Description of change

The form for editing the information for an OMIS quote has been reactified.

## Test instructions

Go to an OMIS order, scroll down to `Information for the quote` and click the Edit button. You should be able to edit both fields. You should also not be able to enter today's date (or any date before this) in the delivery date field.

## Screenshots

### Before

![Screenshot 2023-09-21 at 14 36 42](https://github.com/uktrade/data-hub-frontend/assets/36161814/3509fc6e-cfd5-4023-a387-0a7676e32557)


### After

![Screenshot 2023-09-21 at 14 36 49](https://github.com/uktrade/data-hub-frontend/assets/36161814/16c06ab2-13f9-4ce3-8721-7c9d8e074932)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
